core/src/main/resources/application.yml

```diff
@@ -299,6 +299,7 @@ cb:
         Standard_E4a_v4,
         Standard_E8a_v4,
         Standard_E16a_v4,
+        Standard_E16ds_v4,
```

Comment:
was this discussed with Gaurav Patel ? We need to add these to the pricing sheet

Reply:
Now, this is not required, Let me remove it.

Reply:
No, this is not discussed with Gaurav Patel.
We will require this.